src/Parser/Core/Modules/SpellUsable.js

```diff
@@ -101,7 +101,7 @@ class SpellUsable extends Analyzer {
    * @param {boolean} resetAllCharges Whether all charges should be reset or just the last stack. Does nothing for spells with just 1 stack.
    * @param {number} timestamp Override the timestamp if it may be different from the current timestamp.
    */
-  endCooldown(spellId, resetAllCharges = false, timestamp = this.owner.currentTimestamp) {
+  endCooldown(spellId, resetAllCharges = false, timestamp = this.owner.currentTimestamp, remainingCDR = 0) {
```

Comment:
Please also add remainingCDR to the documentation comment above. For a core module like SpellUsable I think it's important we keep the documentation up to date.